exercises/regularization/ex_rnw/sol_lasso_subdifferential.Rnw

```diff
@@ -16,15 +16,15 @@ plot(xx, yy, type = "l", xlab = "x", ylab = "f(x)", ylim=c(-4,10), col ="red", l
 for(x0 in seq(-2,2,by=0.5))
```

Comment:
oben in Zeile 5 löschen: "and for $x=0$ exactly the function $2x^2$ mirrored at the $f(x) = 0$ axis."

Reply:
updated with last commit fe8dc3a11d71afcc3d170f2e580b71ffba1c691b